packages/donutChart/components/DonutChart.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import {
+  themeBorder,
+  themeBrandPrimary
+} from "../../design-tokens/build/js/designTokens";
+import { tintContentSecondary } from "../../shared/styles/styleUtils/typography/color";
+
+interface DonutChartDatum {
+  percentage: number;
+  color?: React.CSSProperties["stroke"];
+}
+
+export interface DonutChartProps {
+  /**
+   * objects that define the size and color of segments in the donut chart
+   */
+  data: DonutChartDatum[];
+  /**
+   * the primary piece of data in the chart - appears large
+   */
+  label?: string;
+  /**
+   * a caption for the label
+   */
+  text?: string;
+}
+
+class DonutChart extends React.PureComponent<DonutChartProps, {}> {
+  public render() {
+    const { data, label, text } = this.props;
+    const strokeWidth = 2;
+    const radius = 100 / (Math.PI * 2);
+    const diameter = radius * 2 + strokeWidth;
+    const circleCenter = diameter / 2;
+
+    const circleEls = (data: DonutChartDatum[]) =>
```

Comment:
Small nit: could this just be `const circleEls = data.map ...` instead of assigning to a fn and calling it?

Reply:
Yea, that makes way more sense 🤦‍♂